appveyor.yml

```diff
@@ -49,14 +58,20 @@ test_script:
 
     reg import ..\turbodbc\appveyor\odbc-postgresql.reg
 
+    reg import ..\turbodbc\appveyor\odbc-mssql.reg
+
     reg import ..\turbodbc\appveyor\odbc-data-sources.reg
 
     SET PATH=C:\Program Files\PostgreSQL\9.6\bin;C:\Program Files\MySQL\MySQL Server 5.7\bin;%PATH%
 
+    python -c "open('../turbodbc/python/turbodbc_test/mssql.json','w').write(open('../turbodbc/python/turbodbc_test/query_fixtures_mssql.json').read().replace('StrongPassword1','Password12!'))"
```

Comment:
I really don't like this line :-). Either change the MSSQL password to be consistent with the existing test fixture, or change the password listed in the test fixture to be consistent with the MSSQL password.